src/tests/multihost/alltests/test_misc_proxy.py

```diff
@@ -349,6 +351,56 @@ def test_0002_bz1209483(self, multihost, backupsssdconf):
         ssh1.close()
         execute_cmd(multihost, "userdel -rf foo2")
 
+    def test_bz1927195(self, multihost, backupsssdconf):
+        """
+        :title: sssd runs out of proxy child slots and
+          doesn't clear the counter for Active requests
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1927195
+        :id: 4c5b53e2-ec88-11ec-a41a-845cf3eff344
+        :steps:
+            1. Configure sssd with proxy
+            2. Try sssctl user-checks user@domain 10 times
+            3. Logs should not have error 'All available
+               child slots are full'
+        :expectedresults:
+            1. Should succeed
+            2. Should succeed
+            3. Should succeed
+        """
+        # When using authentication provider as proxy,
+        # User authentication suddenly stops working and
+        # starts working again only after restarting the sssd service.
+        tools = sssdTools(multihost.client[0])
+        domain_name = tools.get_domain_section_name()
+        domain_params = {'debug_level': '9',
+                         'id_provider': 'ldap',
+                         'proxy_pam_target': 'proxy_sleep',
+                         'auth_provider': 'proxy',
+                         'chpass_provider': 'ldap',
+                         'proxy_max_children': '10',
+                         'enumerate': 'false',
+                         'entry_cache_timeout': '300',
+                         'access_provider': 'proxy'}
+        tools.sssd_conf('domain/' + domain_name, domain_params)
+        tools.clear_sssd_cache()
+        client = multihost.client[0]
+        file_location = "/10_ssctl.sh"
+        client.transport.put_file(os.path.dirname(os.path.abspath(__file__))
+                                  + file_location, '/tmp/10_ssctl.sh')
+        execute_cmd(multihost, f"chmod 755 /tmp/10_ssctl.sh")
```

Comment:
You do not need the 'f' here since the string is constant, this is causing an issue in the flake8 ci check.

Reply:
Fixed